### ISSUE TITLE
Hex editor should be accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#98715]
Hey Logan, just kreating this so we keep it in mind.
The hex editor should be accessible, that means that all fields should be navigalbe by keyboard and that when you navigate the screen reader always reads the correct content.

For more details we can also sync. Feel free to ping me
